### ISSUE TITLE
List topic in tap to different screen

### ISSUE THREAD

**Issue by TTutorials** [OLayemii/flutter-ui-kits#9]
Hello, nice scripts.

I am new in flutter and try to assigned topic list to different screen. Can you please help?

Big thanks

**Comment by olayemii** [comment]
@TTutorials  are you trying to link  a widget to another screen?

**Comment by TTutorials** [comment]
@olayemii yes, I want to link the first list to widget A and the second list to widget B. The goal is, each list tap will go to the different screen

**Comment by olayemii** [comment]
@TTutorials you need a `Navigator.push()` call

[See here](https://flutter.dev/docs/cookbook/navigation/navigation-basics)
